actionpack/test/controller/redirect_test.rb

```diff
@@ -96,6 +96,16 @@ def redirect_to_url
     redirect_to "http://www.rubyonrails.org/"
   end
 
+  def redirect_to_url_with_to_str
```

Comment:
```suggestion
  def redirect_to_url_with_stringlike
```
When a method defines `#to_str`, it is a string-like object.